Structure_Engine/Query/ReinforcementLayout.cs

```diff
@@ -401,20 +401,20 @@ private static void ExtractInnerAndOuterEdges(ConcreteSection section, out List<
 
         private static List<LongitudinalReinforcement> LongitudinalReinforcement(this ConcreteSection section)
         {
-            if (section == null || section.Reinforcement == null || section.Reinforcement.Count == 0)
+            if (section == null || section.RebarIntent == null || section.RebarIntent.BarReinforcement.IsNullOrEmpty())
```

Comment:
This will raise an error message that was not there before (reinforcement list being empty).

Was this intentional?

Reply:
Will only raise an error message if `BarRebarIntent` is provided and the `Reinforcement` is `null` or empty which would be an edge case and worthy of an error.

The `null` check for the `BarRebarIntent` will capture any default `ConcreteSection` and no error message will be provided as  an empty List<LongitudinalReinforcement>` will be returned.